test/perf/tests/styled-system-spaces/index.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { space } from 'styled-system';
+
+const App = () => {
+  const result = space({
+    theme: {},
+    p: [1, 2, 3],
+  });
+
+  return null;
+}
+
+export default App;
```

Comment:
Where possible, I would encourage to avoid duplicating the name of the component (App)

```suggestion
export default function App() {
  const result = space({
    theme: {},
    p: [1, 2, 3],
  });

  return null;
}
```

Reply:
Will update the names 👍